awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx

```diff
@@ -55,7 +58,7 @@ function WorkflowJobTemplateDetail({ template, i18n, webHookKey }) {
       )}
       {template.webhook_service && (
         <TextListItem component={TextListItemVariants.li}>
-          {i18n._(t`- Webhooks`)}
+          {i18n._(t`- Enable Webhook`)}
```

Comment:
Enable Webhook (and its respective tooltip) should both say "Enable Webhooks". Might as well change the aria-label etc. as well if you do this.

Reply:
Yea this was done intentionally to match the old UI

<img width="1508" alt="Screen Shot 2020-03-04 at 9 47 17 AM" src="https://user-images.githubusercontent.com/9889020/75891019-2f4e6f80-5dfd-11ea-9498-32239fccb218.png">

I would defer to @wenottingham / @tvo318 on this one

Reply:
@mabashian and @dsesami Based on their feedback let's make a follow up issue....might be good to mark it as good first issue also.

Reply:
IMO, I agree with @dsesami- should be plural, since the old one was plural: “Enable Webhooks”

Reply:
https://github.com/ansible/awx/issues/6163